Class-10-(Razor-Views).md

```diff
@@ -0,0 +1,33 @@
+# Class 10: Razor Views
+
+## In the prep work this week, the students learned
+
+1. More about the "view" portion the MVC design pattern.
+1. Why using templates to design a website is a good idea.
+1. How to create templates using Razor.
+1. How to add iteration, conditionals, and fragments to a template.
```

Comment:
I would change "fragments" to "forms". We do discuss partial views in the chapter, but it doesn't receive as much attention as forms.